mix.exs

```diff
@@ -38,9 +38,11 @@ defmodule OAuth2.Mixfile do
 
   defp deps do
     [
-      {:hackney, "~> 1.13"},
+      {:tesla, "~> 1.4"},
+
 
       # Test dependencies
+      {:hackney, "~> 1.17", only: [:dev, :test]},
```

Comment:
```suggestion
      {:tesla, "~> 1.4"},
      {:hackney, "~> 1.17"},

      # Test dependencies
```

I am not sure how to think about this because the default client is `hackney` so technically we depend on it.

So if we remove it, I am not sure how people we would break, and we should think about making clear what will happen and how to fix it!

Reply:
If we use Httpc without any problem

Reply:
Right, maybe that could be the solution, but honestly, I haven't used that client in years; I always swap it since Tesla allowed me to.

Nowadays, `Mint` by default.

Reply:
Yeah, it's really nice being able to use newer Http clients.